addons/web/controllers/main.py

```diff
@@ -1550,6 +1550,8 @@ def from_data(self, fields, rows):
                     cell_style = datetime_style
                 elif isinstance(cell_value, datetime.date):
                     cell_style = date_style
+                elif type(cell_value) in [list, tuple]:
```

Comment:
why not use `isinstance` as above?

```suggestion
                elif isinstance(cell_value, (list, tuple)):
```

Reply:
I've changed it.
I didn't know I could use a tuple for the second param of `isinstance`. :)